endpoints.json

```diff
@@ -88,5 +88,25 @@
 				"created_at": "2020-07-09T20:11:00.000Z"
 			}
 		}
-	}
+	},
+  "PATCH /api/articles/:article_id" :{
+    "description": "updates votes property of the given article based on article_id and returns updated article object",
+		"queries": [],
+    "params": ["article_id", "req.body"],
```

Comment:
article_id and, req.body are not params of this endpoint. If you are unsure what I mean by this I would recommend looking at the notes for what params are.